src/monitor-pipeline-group/azext_monitor_pipeline_group/tests/latest/test_monitor_pipeline_group.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.testsdk import *
+
+
+class MonitorPipelineGroupScenario(ScenarioTest):
+    # TODO: add tests here
```

Comment:
Please add tests for your added cmds.

Reply:
Added test_monitor_pipeline_group to cover create/list/show/update/delete commands.